android/ta_player_plugin/build.gradle

```diff
@@ -64,6 +68,11 @@ dependencies {
             exclude group: 'com.google.android.exoplayer', module: 'exoplayer-smoothstreaming'
         }
     }
+//    implementation project(':cast_plugin')
+    implementation("com.applicaster:GenericChromecast-Android-no-meta:1.0.+") {
```

Comment:
Why do we need a dependency on the plugin here?

Reply:
Yep, you're right.